aiidalab_widgets_base/computational_resources.py

```diff
@@ -702,10 +702,19 @@ def __init__(self, **kwargs):
             style=STYLE,
         )
 
+        # Memory per node.
+        self.default_memory_per_machine = ipw.IntText(
+            value=16000000,
+            step=1000000,
+            description="#Memory(RAM) per node (kB):",
```

Comment:
I don't think it's a good idea to ask the user to provide the value in kB since the number of machines providing RAM on the order of kB should at this point be restricted to edge and embedded device that are not typically known to run AiiDA or Jupyter. You can always internally convert the value if that is what is expected by the code plugin.